models/Character.js

```diff
@@ -42,7 +42,7 @@ var CharacterSchema = new mongoose.Schema({
     type: mongoose.Schema.Types.ObjectId,
     ref: 'CharacterClass'//,
 //  required: true
-  }  
+  }  */
```

Comment:
Why should this be commented out from the model?

Reply:
I dit it under testing as I tried to solve a problem I had and forgot to change it. I only just noticed it when creating the pull request and figured it could be changed with the implementation of races and characterClasses which are still to come.

Reply:
I see, but this means we now have "half" support for it. We treat races and classes in the creation (though we do not save them), but are not able to return them. Is it still causing problems if it's uncommented?

Reply:
Technically we do not support them at all, I know what you mean that the code is there some places and not others, but we can not create them and therefore not add them to the character.
But no, it was not the cause of the problem

Reply:
I can just push a version with it uncommented if you want ;)

Reply:
I did it in my new commits ;)
I also ensured that they work (see the updated README.